### PR TITLE
Segments - Documentation for date based + text filters

### PR DESCRIPTION
**Pull request by Maxell92**
Segments - Documentation for date based + text filters.

Related to Segment refactoring: https://github.com/mautic/mautic/pull/5755

### CODE REVIEW

File: en/contacts/manage_segments.md [hunk 1]
```diff
@@ -33,6 +33,42 @@ result as the intersection of the subsets. You can then manipulate the contacts
 
 ![](/contacts/media/common-leads-in-segments.jpg)
 
+#### Date options
+
+Date filters allow you to choose a date via DatePicker:
+
+![](/contacts/media/segment-filters-datepicker.png)
+
+Hovewer you can specify much more here. Mautic recognizes relative formats too (these string are not transatable):
```
Comment (dbhurley) [inline en/contacts/manage_segments.md]: translatable

File: en/contacts/manage_segments.md [hunk 2]
```diff
@@ -33,6 +33,42 @@ result as the intersection of the subsets. You can then manipulate the contacts
...
+
+* `+1 day` (you can use `1 day` too)
```
Comment (dbhurley) [inline en/contacts/manage_segments.md]: (you can also use `1 day`)

File: en/contacts/manage_segments.md [hunk 3]
```diff
@@ -33,6 +33,42 @@ result as the intersection of the subsets. You can then manipulate the contacts
...
+* `+1 year` / `-2 years` / `3 years ago`
+
+Example (Consider that today is `2018-03-02`):
+* `Date identified equals -1 week` returns all contacts identified on 2018-02-23.
+* `Date identified less than -1 week` returns all contacts identified before 2018-02-23.
+* `Date identified equals -1 months` returns all contacts identified on 2018-02-02.
+* `Date identified greater or equal -1 year` returns all contacts identified 2017-03-02 and after.
+* `Date identified greater than -1 year` returns all contacts identified after 2017-03-02.
+
+Beside this you can specify your date with text. These formulas are **transatable**, so make sure you use them in correct format.
```
Comment (dbhurley) [inline en/contacts/manage_segments.md]: translatable

**Comment by Maxell92** [comment]
@dbhurley Updated. Thank you very much!

**Pull request merged**